### ISSUE TITLE
Mistake in Polish layout

### ISSUE THREAD

**Issue by MrBober** [florisboard/florisboard#1960]
Default accent letter for `z` should be `ż`, not `ź`.

**Comment by pgronkievitz** [comment]
according to [Wikipedia](https://pl.wikipedia.org/wiki/Alfabet_polski#Cz%C4%99sto%C5%9B%C4%87_wyst%C4%99powania_liter) `ż` is over order of magnitude more common than `ź`, so it just makes sense to swap those two and make `ż` faster to type

**Comment by pgronkievitz** [comment]
I think it should be enough to just swap `ź` and `ż` like that (filenames irrelevant in this case as I didn't clone whole repository)
```diff
diff --git a/pl.json b/pl_new.json
index dc45da2..63a27bc 100644
--- a/pl.json
+++ b/pl_new.json
@@ -43,8 +43,8 @@
     },
     "z": {
       "relevant": [
-        { "$": "auto_text_key", "code":  378, "label": "ź" },
-        { "$": "auto_text_key", "code":  380, "label": "ż" }
+        { "$": "auto_text_key", "code":  380, "label": "ż" },
+        { "$": "auto_text_key", "code":  378, "label": "ź" }
       ]
     },
     "~right": {
```

**Comment by patrickgold** [comment]
Above PR fixes the popup ordering of z, `ż` is now prioritized over `ź`. Will be released in 0.4.0-alpha02
